docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,139 @@ description: Dagster can orchestrate dbt alongside other technologies.
+# If DAGSTER_DBT_PARSE_PROJECT_ON_LOAD is set, a manifest will be created at runtime.
+# Otherwise, we expect a manifest to be present in the project's target directory.
+if os.getenv("DAGSTER_DBT_PARSE_PROJECT_ON_LOAD"):
+    dbt_parse_invocation = dbt.cli(["parse"], manifest={}).wait()
+    dbt_manifest_path = dbt_parse_invocation.target_path.joinpath("manifest.json")
+else:
+    dbt_manifest_path = dbt_project_dir.joinpath("target", "manifest.json")
+```
 
-2. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the assets:
+As the comment explains, the code gives you a choice about how to create this `manifest.json` file. Based on the `DAGSTER_DBT_PARSE_PROJECT_ON_LOAD` environment variable, either:
 
-   <Image
-   alt="Asset graph in the UI, containing dbt models loaded as Dagster assets"
-   src="/images/integrations/dbt/using-dbt-with-dagster/load-dbt-models-dbt-asset-graph.png"
-   width={1090}
-   height={894}
-   />
+1. **This code generates the `manifest.json` for you.** This is the easiest option during development, because you never need to worry about the file being out-of-date with your dbt project. But generating the file can be time-consuming when you have a large dbt project, so it's generally recommended to go with the second option when you're deploying your dbt project to production.
+2. **This code leaves it up to you to generate the `manifest.json` file on your own, and this code just reads it.** This option allows you to only generate it once every time you deploy, rather than every time the code in this module gets executed.
+
+In this tutorial, we're going with the first option: When we ran `dagster dev`, we set the `DAGSTER_DBT_PARSE_PROJECT_ON_LOAD` environment variable.
+
+Once you've got a path to a `manifest.json`, it's time to define your Dagster assets using it. The following code, also already in your project's `definitions.py`, does this:
+
+```python file=integrations/dbt/tutorial/load_dbt_models/definitions.py startafter=start_dbt_assets endbefore=end_dbt_assets
+@dbt_assets(manifest=dbt_manifest_path)
+def jaffle_shop_dbt_assets(context: OpExecutionContext, dbt: DbtCliResource):
+    yield from dbt.cli(["build"], context=context).stream()
+```
+
+This code might look a bit fancy, because it uses a decorator. Here's a breakdown of what's going on:
+
+- It creates a variable named `jaffle_shop_dbt_assets` that holds an object that represents a set of Dagster assets.
+- These Dagster assets reflect the dbt models described in the manifest file. The manifest file is passed in using the `manifest` argument.
+- The decorated function defines what should happen when you "materialize" one of these Dagster assets, e.g. by clicking the **Materialize** button in the UI or materializing it automatically by putting it on a schedule. In this case, it will invoke the `dbt build` command on the selected assets. The `context` parameter that's provided along with `dbt build` carries the selection.
```

Comment:
Does there need to be quotes around "materialize"?

Reply:
nope